plaid-stl/src/npm/shared_structs/mod.rs

```diff
@@ -169,7 +168,7 @@ pub struct NpmToken {
     pub token_name: Option<String>,
     pub token_type: Option<String>,
     // Deserialize the date field from the ISO 8601 format
-    #[serde(default, deserialize_with = "deserialize_option_timestamp")]
+    #[serde(with = "chrono::serde::ts_seconds_option")]
```

Comment:
@michelemin  `chrono` has a built in function for this. Do you bless this or should I leave your portion as is?